src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,90 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int INITIAL_CAPACITY = 16;
+    private static final float LOAD_FACTOR = 0.75f;
+    private int size;
+    private Node<K, V>[] hashTable;
+
+    public MyHashMap() {
+        hashTable = new Node[INITIAL_CAPACITY];
+    }
 
     @Override
     public void put(K key, V value) {
-
+        if (size >= getThreshold()) {
+            resize();
+        }
+        int index = hashKey(key);
+        Node<K, V> node = hashTable[index];
+        while (node != null) {
+            if (Objects.equals(key, node.key)) {
```

Comment:
Avoid using getters and setters in the Node class. Since Node is a private inner class of MyHashMap, you can directly access its fields without getters and setters.